README.md

```diff
@@ -66,4 +91,53 @@ WifiManager.getCurrentWifiSSID()
 	console.log('Cannot get current SSID!')
 })
 ```
-  
\ No newline at end of file
+
+## API
+
+_The api documentation is in progress._
+
+### connectToProtectedSSID(SSID: string, password: string, isWep: boolean): Promise
+
+Returns a promise that resolves when connected or rejects with the error when it couldn't connect to the wifi network.
+
+#### SSID
+Type: `string`
+The SSID of the wifi network to connect with.
+
+#### password
+Type: `string`
+The password of the wifi network to connect with.
+
+#### isWeb
+Type: `boolean`
+Used on iOS. FIXME: why?
```

Comment:
isWep*
Look in doc not has interface with passphrase and without isWep
https://developer.apple.com/documentation/networkextension/nehotspotconfiguration

Reply:
Tbh I would not support connect to Wep networks at all :-D. They can be hacked in seconds.